test/app/features/appstate/domain/usecases/app_preferences_usecase_test.dart

```diff
@@ -219,10 +252,9 @@ void main() {
       );
 
       expect(value.isLeft(), true);
-      expect(
-        value.swap().getOrElse(() => null),
-        InactivityError.customerActive,
-      );
+      value.swap().forEach((error) {
+        expect(error, InactivityError.customerActive);
+      });
```

Comment:
```suggestion
      expect(
        value.fold((l) => l, (r) => r),
        InactivityError.customerActive,
      );
```

Acredito que utilizar o `.fold` facilita a leitura do teste.

Reply:
feito